lib/rucio/tests/temp_factories.py

```diff
@@ -68,7 +72,7 @@ def _cleanup_rses(self):
             # So running test in parallel results in some tests failing on foreign key errors.
             rse_core.del_rse(rse_id)
 
-    def _make_rse(self, scheme, protocol_impl, parameters=None, add_rse_kwargs=None):
+    def _make_rse(self, scheme, protocol_impl, parameters=None, add_rse_kwargs=None) -> "Tuple[str, str]":
```

Comment:
Does this work as expected? Tuple in a quotation marks? Am I missing something obvious?

Reply:
relevant [PEP 3107](https://peps.python.org/pep-3107/)
more [info](https://www.geeksforgeeks.org/function-annotations-python/)